### ISSUE TITLE
Setting ReCaptcha v3 score threshold  

### ISSUE THREAD

**Issue by sJones1997** [GawainLynch/bolt-extension-boltforms#310]
Have been digging around the docs looking for how to set the score threshold for ReCaptcha v3 but can't find anything.

Is there a way to handle this?

**Comment by bobdenotter** [comment]
@sJones1997 Which version of Bolt / Forms?

**Comment by sJones1997** [comment]
@bobdenotter  "bolt/forms": "^1.4" is what's in the composer.json

**Comment by bobdenotter** [comment]
I don't think it's supported for the legacy version of Bolt. You could consider updating to Bolt 5.

**Comment by sJones1997** [comment]
Might be getting packages mixed up but currently have "bolt/core": "^5.0" in composer.json.

**Comment by sJones1997** [comment]
@bobdenotter I did get packages mixed up, this original issue was for the bolt/forms repo.
I have cloned the bolt/forms repo, made a local branch and added in a feature that allows users to specify a Recaptcha v3 threshold. Can I make a PR for the feature?

**Comment by bobdenotter** [comment]
Yes, looking at it, i don't think the threshold is implemented yet.. If you would like to make a PR for that, that would be very much appreciated. :-)
